### ISSUE TITLE
[Request] Video Overlay Button for Long Form Videos

### ISSUE THREAD

**Issue by mmupmip** [YTLitePlus/YTLitePlus#333]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you use YouTube Premium?

❌ No, I don't use YouTube Premium

### My feature request

There doesn't seem to have a button overlay in the video player and it would be nice to have one. Also, currently there are no options for audio only download. 
The shorts download automatically saves into Photos for some, and prompts in Files for others (due to size?). However, I read somewhere else that the 5.0 update will prompt storage location for everything so this would probably get fixed too?

**Comment by bhackel** [comment]
Yes I believe that v5 will also add the in-player download button. Once it is out, we will be able to handle bugs better.
